apps/web/src/components/Feed/PostItem.tsx

```diff
@@ -55,9 +61,18 @@ export function PostItem({
 
   const isDesktop = useIsDesktopWindowWidth();
 
-  if (!isDesktop) {
+  const useVerticalLayout = !isDesktop || bigScreenMode;
+
+  const reportError = useReportError();
+
+  if (!post.author) {
+    reportError('Post author is undefined', { tags: { postId: post.dbid } });
+    return null;
+  }
```

Comment:
we want to make sure to avoid calling `reportError` directly in the render function

here we can just `throw` instead, since everywhere the `PostItem` component is rendered is already wrapped in a Suspense boundary that will handle it:

```ts
  if (!post.author) {
    throw new ErrorWithSentryMetadata('Post author is undefined', { postId: post.dbid }); // no need to specify `tags`
  }
```

Reply:
ooo perfect ok good call! ty